src/snapred/backend/data/LocalDataService.py

```diff
@@ -57,7 +57,7 @@
 
 
 def version_pattern(x: int) -> str:
-    return wnvf.formatVersion(x, wnvf.vPrefix.FILE)
+    return f"v_{wnvf.formatVersion(x, use_v_prefix=wnvf.vPrefix.FILE)}"
```

Comment:
I'm wondering if instead this should live in the valueformatter

Reply:
I added this here because writing out 
``` python
wnvf.formatVersion(version, use_v_prefix=False)
```
every time I needed to format a file name was really tedious.  I also imported this a lot into the unit tests, to help with versioning consistency

Reply:
I can add an additional formatter exclusive to files, that should save the same purpose